akka-actor/src/main/scala/akka/util/ByteString.scala

```diff
@@ -528,7 +528,7 @@ object CompactByteString {
     else {
       val ar = new Array[Byte](bytes.size)
       bytes.copyToArray(ar)
-      CompactByteString(ar)
+      ByteString.ByteString1C(ar)
```

Comment:
There is a problem - `ByteString1C` doesn't clone array, it just stores ref in field. So, if using `toArray` we'll allow creating implicitly mutable ByteString by calling `ByteString(someMutableArray: _*)`.